test.js

```diff
@@ -147,3 +147,20 @@ test('supports localhost URLs', t => {
 	t.is(linkifyUrls('https://localhost'), '<a href="https://localhost">https://localhost</a>');
 	t.is(linkifyUrls('https://localhost/foo/bar'), '<a href="https://localhost/foo/bar">https://localhost/foo/bar</a>');
 });
+
+test('skips truncated URLs', t => {
+	t.is(linkifyUrls('https://github.com/sindresorhus/linkify-…'), 'https://github.com/sindresorhus/linkify-…');
+	t.is(linkifyUrls('https://github.com/sindresorhus/linkify-… and https://github.com/sindresorhus/linkify-…'), '<a href="https://github.com/sindresorhus/linkify-">https://github.com/sindresorhus/linkify-</a>… and https://github.com/sindresorhus/linkify-…');
+	t.is(
+		html(linkifyUrls('See https://github.com/sindresorhus/linkify-urls… and https://github.com/sindresorhus/linkify-…', {
+			type: 'dom',
+		})),
+		html(domify('See <a href="https://github.com/sindresorhus/linkify-urls">https://github.com/sindresorhus/linkify-urls</a>… and https://github.com/sindresorhus/linkify-…')),
```

Comment:
It doesn't look like it's working. Neither one should be linkified.

Reply:
I wasn't sure how to handle that case, websites usually do not truncate in the middle of a text body.

Reply:
Arguments:

- Hacker news is such example, URLs are truncated anywhere they appear (but this happens as a consequence of linkification)
- If `https://example.com/…` is skipped, then `https://example.com/… for example` should probably be skipped too

I'm not confident about this though. @sindresorhus?

Reply:
> If https://example.com/… is skipped, then https://example.com/… for example should probably be skipped too

Agreed